docs/topics/appendix-s2i-build-process.adoc

```diff
@@ -1,16 +1,16 @@
 [[s2i-build-process]]
 = The Source-to-Image (S2I) Build Process
-link:https://docs.openshift.com/container-platform/3.4/architecture/core_concepts/builds_and_image_streams.html#source-build[Source-to-Image] (S2I) is a build tool for generating reproducible Docker-formatted container images from online SCM repositories with application sources.
+link:https://docs.openshift.com/container-platform/latest/architecture/core_concepts/builds_and_image_streams.html#source-build[Source-to-Image] (S2I) is a build tool for generating reproducible Docker-formatted container images from online SCM repositories with application sources.
```

Comment:
Why do you say "Docker-formatted container" instead of "Docker container image" ?

Reply:
@cmoulliard I think we used  the definition for Source-to-Image (S2I) Build from the architecture documents.  https://docs.openshift.com/container-platform/latest/architecture/core_concepts/builds_and_image_streams.html#source-build .

Reply:
@cmoulliard …to add onto this, OpenShift's "Docker-formatted container image" terminology comes from the _Container Technology Language and Reference Guide_ produced by Product Marketing.

Reply:
this is per the guidelines:  "container image" or "docker-formatted container ".